compiler/rustc_codegen_ssa/Cargo.toml

```diff
@@ -35,6 +35,6 @@ rustc_target = { path = "../rustc_target" }
 rustc_session = { path = "../rustc_session" }
 
 [dependencies.object]
-version = "0.22.0"
+version = "0.24.0"
 default-features = false
-features = ["read_core", "elf", "macho", "pe", "unaligned", "archive"]
+features = ["read_core", "elf", "macho", "pe", "unaligned", "archive", "write"]
```

Comment:
Once the object dependency of backtrace is updated, this new feature will become unified with libstd resulting in binary bloat. Should we switch rustc to the version 2 resolver already?

Reply:
No libstd and rustc have separate builds, so they get separate feature sets. This won't affect libstd. Changing the resolver for Cargo also won't affect anything here.